### DISCUSSION TITLE
Need KM hashes for insecure MSI hardware

### DISCUSSION THREAD

**Discussion by hughsie** [fwupd/fwupd#5797] [General]
I'd like to collect what READ_FILE_EX FILE_IDs are being used here. I've asked people to contribute data [here](https://blogs.gnome.org/hughsie/2023/05/09/msi-and-insecure-kms/).

**Comment by evgeni** [comment]
As discussed in https://chaos.social/@zhenech/110340146794557654, here is the data from a (unaffected) MSI H110I PRO (with an i5-6500, running CentOS Stream 8) that shows `OEM Public Key Hash ME 0000000000000000000000000000000000000000000000000000000000000000`.
[evgeni-meinfo.txt](https://github.com/fwupd/fwupd/files/11443723/evgeni-meinfo.txt)
[evgeni-meinfo-strace.txt](https://github.com/fwupd/fwupd/files/11443725/evgeni-meinfo-strace.txt)
